### ISSUE TITLE
Unable to load project.

### ISSUE THREAD

**Issue by mdmnbg** [stackblitz/bolt.new#1543]
### Describe the bug

I had to reload the page cause i was getting a token error (the prompt was over 2000000 characters or something like that) and then suddently i'm facing "Unexpected token 'U', "Unexpected"... is not valid JSON
An unknown error occured while running an action."

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-6b9fx1

### Steps to reproduce

1. Try to reload the page after a token error 2. face the issue...

### Expected behavior

to be able to go back to my project

### Screen Recording / Screenshot

<img width="477" alt="Screenshot 2024-11-04 at 21 44 00" src="https://github.com/user-attachments/assets/d9eb4150-31bd-431a-885a-2f5db3ff5c4a">

### Platform

```
Browser name = Chrome
Full version = 130.0.0.0
Major version = 130
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 52497574,
  "usedJSHeapSize": 46455898,
  "jsHeapSizeLimit": 4294705152
}
Username = pvlvl
Chat ID = 1632bea242ea
```

### Additional context

Error:
```
An unknown error occured while running an action.
```

**Comment by matthewmarleyuk** [comment]
Im having the same issue, none of my projects are loading at all

**Comment by shirazh7** [comment]
Same here

**Comment by lancejia24** [comment]
this happen after i pay again

**Comment by kc0tlh** [comment]
@mdmnbg @lancejia24 @shirazh7 @matthewmarleyuk Thanks for the report and apologies for the frustrating experience! The fix is explained in #1471
